openghg/store/_emissions.py

```diff
@@ -95,17 +106,42 @@ def read_file(
                 - suitable pandas Offset Alias
                 - tuple of (value, unit) as would be passed to pandas.Timedelta function
             continuous: Whether time stamps have to be continuous.
-            overwrite: Should this data overwrite currently stored data.
+            if_exists: What to do if existing data is present.
+                - "default" - checks new and current data for timeseries overlap
+                   - adds data if no overlap
+                   - raises DataOverlapError if there is an overlap
+                - "new" - just include new data and ignore previous
+                - "replace" - replace and insert new data into current timeseries
+            save_current: Whether to save data in current form and create a new version.
+                If None, this will depend on if_exists input ("default" -> True), (other -> False)
+            overwrite: Deprecated. This will use options for if_exists="new" and save_current=True.
+            force: Force adding of data even if this is identical to data stored.
         Returns:
             dict: Dictionary of datasource UUIDs data assigned to
         """
         from openghg.types import EmissionsTypes
-        from openghg.util import clean_string, hash_file, load_emissions_parser
+        from openghg.util import (
+            clean_string,
+            hash_file,
+            load_emissions_parser,
+            check_if_need_new_version,
+        )
 
         species = clean_string(species)
         source = clean_string(source)
         domain = clean_string(domain)
 
+        if overwrite and if_exists == "default":
+            logger.warning("Overwrite flag is deprecated in preference to `if_exists` (and `save_current`) inputs."
+                           "See documentation for details of these inputs and options.")
+            if_exists = "new"
+
+        # Making sure data can be force overwritten if force keyword is included.
+        if force and if_exists == "default":
+            if_exists = "new"
+
+        new_version = check_if_need_new_version(if_exists, save_current)
```

Comment:
```suggestion
        new_version = check_if_need_new_version(if_exists=if_exists, save_current=save_current)
```

Reply:
Yeah all good, we don't need to be too pedantic about it for functions with < 3 arguments.